packages/connectivity/src/scp-cf/destination/service-binding-to-destination.ts

```diff
@@ -21,10 +27,48 @@ export const serviceToDestinationTransformers: Record<
   workflow: workflowBindingToDestination,
   'service-manager': serviceManagerBindingToDestination,
   xsuaa: xsuaaToDestination,
-  aicore: aicoreToDestiation
+  aicore: aicoreToDestination
 };
 
-async function aicoreToDestiation(
+/**
+ * Convenience function to create an OAuth2ClientCredentials destination from the provided service binding.
+ * If a JWT is provided as part of options, the tenant in the JWT is used for client credentials grant, else the provider tenant is used.
+ * Supported service types are:
+ * - business-logging
+ * - destination
+ * - s4-hana-cloud (falls back to basic authentication for this service type)
+ * - saas-registry
+ * - workflow
+ * - service-manager
+ * - xsuaa
+ * - aicore.
+ *
+ * Throws an error if the provided service binding is not supported.
+ * @param serviceBinding - The service binding to transform.
+ * @param options - Options used during fetching destinations;includes a JWT payload {@link JwtPayload} and options to influence caching behavior see {@link CachingOptions}.
```

Comment:
[pp] I think we can be less verbose here. The type `ServiceBindingTransformOptions` is linked automatically and it includes the subtypes.
```suggestion
 * @param options - Options used for fetching the destination.
```